### PR TITLE
Add cluster Labels at installation time.

### PR DESCRIPTION
**Pull request by Andreagit97**
# Description

This new feature allows user to specify labels for his clusters at Liqo installation with helm chart.

### CODE REVIEW

File: deployments/liqo/values.yaml [hunk 1]
```diff
@@ -98,6 +98,8 @@ discovery:
   config:
     # -- Set a mnemonic name for your cluster
     clusterName: ""
+    # -- Set labels that must be exported in order to select your specific cluster among the others
```
[the remark this answers is not in the document]
Comment (Andreagit97) [reply deployments/liqo/values.yaml:101]: Wonderful description, thank you so much.

File: deployments/liqo/values.yaml [hunk 2]
```diff
@@ -98,6 +98,9 @@ discovery:
   config:
     # -- Set a mnemonic name for your cluster
     clusterName: ""
+    # -- A set of labels which characterizes the local cluster when exposed remotely as a virtual node.
+    # It is suggested to specify the distinguishing characteristics that may be used to decide whether to offload pods on this cluster.
+    clusterLabels: {} # { "topology.kubernetes.io/zone": us-east-1 , "liqo.io/provider": your-provider }
```
Comment (giorio94) [inline deployments/liqo/values.yaml:103]: For the sake of clarity, I would move the examples on a new line below, one label per line

**Review by giorio94 — APPROVED** [review]
/lgtm

File: deployments/liqo/values.yaml (hunk 2 above)
Comment (Andreagit97) [reply deployments/liqo/values.yaml:103]: yes, sure, thank you.

**Pull request merged**